pkg/standalone/standalone.go

```diff
@@ -307,6 +308,11 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		}
 		print.InfoStatusEvent(os.Stdout, "Use `%s ps` to check running containers.", runtimeCmd)
 	}
+	// TODO: remove below method when components-path flag is removed.
+	err = moveFilesFromComponentsToResourcesDir(DefaultComponentsDirPath(), DefaultResourcesDirPath())
```

Comment:
Since this is init, a standard set of files will be setup in the components-path. Deleting the path should be sufficient. is there a need to copy files from old dir to new dir for init?

Reply:
@akhilac1 we need the existing components to exist in `resources` directory. If the user modified anything beyond the standard set of files in `components`, it will be lost with a simple delete and fresh init.